crates/re_viewport/src/viewport_blueprint.rs

```diff
@@ -291,7 +309,8 @@ impl ViewportBlueprint {
         &self,
         space_views: impl Iterator<Item = SpaceViewBlueprint>,
         ctx: &ViewerContext<'_>,
-        tree_actions: &mut TreeActions,
+        parent_container: Option<egui_tiles::TileId>,
+        focus_tab: bool,
```

Comment:
Rather than having a bool here I think it might be cleaner to return the list of `new_ids` and then let the caller call `focus_tab` on it themselves in the one case where `focus_tab` isn't false.

Reply:
That makes sense!